tests/fakes.py

```diff
@@ -68,3 +81,47 @@
 echo -- "$@" >> "${log_path}/apt_get.args"
 env >> "${log_path}/apt_get.env"
 """
+
+LIVEPATCH_ENABLED_STATUS = """
+cat <<EOF
+client-version: "7.23"
+status:
+- kernel: 4.4.0-87.110-generic
+  running: true
+  livepatch:
+    checkState: {}
```

Comment:
please use named subtitutions, e.g.:

```
checkState: {check_state}
```
so that the .format() call is a bit more explicit (see below)